### PR TITLE
Added new example ja3 fingerprint

### PR DESCRIPTION
**Pull request by vpanchak**
Initial commit - JA3 fingerprint for EdgeWorkers.

### CODE REVIEW

File: edgecompute/examples/authentication/ja3-fingerprinting/main.js [hunk 1]
```diff
@@ -0,0 +1,553 @@
...
+
+  return [protocol_version, cipher_suites_joined, extensions_joined, elliptic_curves_joined, ec_point_formats_joined].join(',');
+}
+
+function getJA3Fingerprint(buffer) {
+  const JA3_input = getJA3RawString(buffer);
+  return md5(JA3_input);
+}
+
+function onClientRequest(request) {
```
[the remark this answers is not in the document]
Comment (iwanaga) [reply edgecompute/examples/authentication/ja3-fingerprinting/main.js:543]: They are separated in original code base. I'll update this branch soon.

File: edgecompute/examples/authentication/ja3-fingerprinting/main.js [hunk 2]
```diff
@@ -0,0 +1,553 @@
...
+  const client_hello = request.getVariable('PMUSER_TLS_CLIENT_HELLO');
+  const buffer = base64toUint8Array(client_hello);
+
+  const JA3_fingerprint = getJA3Fingerprint(buffer);
+  request.setVariable('PMUSER_JA3_FINGERPRINT', JA3_fingerprint);
+
+  return JA3_fingerprint;
```
[the remark this answers is not in the document]
Comment (iwanaga) [reply edgecompute/examples/authentication/ja3-fingerprinting/main.js:550]: This is to make unit testing by Jest easy.
The return value is not used in EdgeWorkers.

File: edgecompute/examples/authentication/.DS_Store
Comment (iwanaga) [inline edgecompute/examples/authentication/.DS_Store:1]: This file should be removed.

File: edgecompute/examples/authentication/ja3-fingerprinting/main.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (iwanaga) [reply edgecompute/examples/authentication/ja3-fingerprinting/main.js:543]: Oh, I don't have permission to push to this PR.
Update will be made by @vpanchak .

[the remark this answers is not in the document]
Comment (vpanchak) [reply edgecompute/examples/authentication/ja3-fingerprinting/main.js]: separated in original code base

**Review by bmatthew — APPROVED** [review]

**Pull request merged**